core/src/test/scala/unit/kafka/admin/AclCommandTest.scala

```diff
@@ -248,7 +289,9 @@ class AclCommandTest extends ZooKeeperTestHarness with Logging {
 
   private def testRemove(cmdArgs: Array[String], resources: Set[ResourcePattern], resourceCmd: Array[String]): Unit = {
     for (resource <- resources) {
-      AclCommand.main(cmdArgs ++ resourceCmd :+ "--remove" :+ "--force")
+      val (out, err) = callMain(cmdArgs ++ resourceCmd :+ "--remove" :+ "--force")
+      Assert.assertEquals("", out)
+      Assert.assertEquals("", err)
```

Comment:
I wonder if it is wise to have the assertions here because `--remove` lists the ACLs. At the moment, it seems that we always use this helper to remove all ACLs so it seems to work OK. Is it always the case?

btw, it seems that we could potentially put the call to `--remove` outside of the loop, no?

Reply:
> At the moment, it seems that we always use this helper to remove all ACLs so it seems to work OK. Is it always the case?

Yes, `testRemove()` is currently always paired with `--add`. Of course that might not always be the case. I guess this issue would go aware if we removed the output checking here and had a dedicated test for it (assuming my interpretation of your other comment is correct).